### PR TITLE
Turn off zeroUnits optimization of css-clean

### LINKED ISSUE #18144: Follow-up on clean-css zeroUnits TODO
Refs https://github.com/twbs/bootstrap/pull/18081#issuecomment-152097011
Refs https://github.com/twbs/bootstrap/issues/14837#issuecomment-153325181

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #18144

### CODE REVIEW

**Comment by cvrebert** [comment]
That doesn't seem to be the correct way to pass that option. I still get
```css
.table-responsive{display:block;width:100%;min-height:0;overflow-x:auto}
```
with `clean-css@3.4.6`. Note the `min-height:0;` in the output.

**Comment by kkirsche** [comment]
@cvrebert should be fixed. I misread `'[+-]properties.zeroUnits' - turn on / off units removal after a 0 value`

File: Gruntfile.js [hunk 1]
```diff
@@ -261,13 +261,14 @@ module.exports = function (grunt) {
...
-        // TODO: disable `zeroUnits` optimization once clean-css 3.2 is released
-        //    and then simplify the fix for https://github.com/twbs/bootstrap/issues/14837 accordingly
         compatibility: 'ie9',
         keepSpecialComments: '*',
         sourceMap: true,
         noAdvanced: true
       },
+      properties: {
+        zeroUnits: false
+      }
```
Comment (cvrebert) [inline Gruntfile.js]: Syntax error; missing comma. Did you test this?

**Comment by kkirsche** [comment]
@cvrebert actually after review, it was added to `clean-css` it seems but was never exposed by `grunt-contrib-cssmin`

**Comment by kkirsche** [comment]
This can't be a fix since neither grunt-contrib-cssmin or clean-css are actually used by Bootstrap...

**Comment by cvrebert** [comment]
WTF are you talking about: https://github.com/twbs/bootstrap/blob/33103c0b522a385e849a4e95682705ffaf5cf86d/package.json#L43

**Comment by cvrebert** [comment]
Feel free to give this another try, but please verify that `min-height:0%;` actually gets output before (re)opening, so as to avoid time wastage.

**Comment by kkirsche** [comment]
yeah I found the issue. it's not exposed from grunt-contrib-cssmin since it's a property not an option

**Comment by kkirsche** [comment]
so that needs an update before anything can be done (tested against V3 branch since only branch I can get to build correctly)

**Comment by kkirsche** [comment]
@cvrebert gruntjs/grunt-contrib-cssmin#247

**Pull request closed without merging**